src/Components/CollectionListItem/ListItem.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+function ListItem({ text, number, color }) {
```

Comment:
make it that the item takes a "type" and number as input instead of text and color and number
where "type" is an object from the collections constant file

@osama please tell me what do you think?

Reply:
Yes, refer to this #43 for more details about the collections types in the constants since they are the same types, so you will only need to pass the text and the type
```suggestion
function ListItem({ number, type }) {
```
and then basically
const type = collections.find((collection) => collection.id === type.id);
// use the type in then, it has the title and color

Reply:
i guess here we can pass the whole type as input because in the collection page we will map over types
so we wont need to use "find" right ?

Reply:
No, we will still need to do the same since we are going to pass only the type which has an `id` field only, the idea is to not have the type fields stored all in the collection object but rather only the id of the type so we can get it from the collections array